### DISCUSSION TITLE
Glitch repeated after 45mm in Y approx. Overlapping lines

### DISCUSSION THREAD

**Discussion by javapast** [arkypita/LaserGRBL#2423] [Q&A]
Hi there,

I have been working for months with no problem, but suddently this started to happen (phot attached). Not idea if the software is missing lines or it repeats the next line/s in the same spot. The weird thing is that it happens only in lines arround 45mm in Y of the image. I am using a sculfun S30 and this started arround 1 month ago, not sure it it was with a new software update. Is it possible to get previous versions? I cannot find them

Thank you!

![20241122_224312_1](https://github.com/user-attachments/assets/3bddefa2-ac54-4af5-b817-b4b42bd2c6f4)
![20241122_224237_1](https://github.com/user-attachments/assets/adc9a814-f3ee-46c7-97e9-b5aa2a927f44)

**Comment by Hostaler** [comment]
You should check the tension of the belts and if you can send an image so we know where the 0 of the X and Y coordinates are on your machine.
Also check if the belt in that Y dimension has dirt between the teeth.
